packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -461,3 +487,78 @@ class DefaultTextEditingShortcuts extends StatelessWidget {
     );
   }
 }
+
+// These constants are based on NSStandardKeyBindingResponding method names
+final Map<String, Intent> _stringToIntent = <String, Intent>{
```

Comment:
I have a WIP pull request before we went with the `isComposing` hack, it's a bit more ambitious than just sending Intents from macOS: https://github.com/flutter/flutter/pull/94961. I'm still not sure if it's a good idea to turn other commands into intents, we can give this a try first.

Reply:
The selector names are specific to iOS/macOS. The method call name `TextInputClient.performIntent` seems to be a little bit too generic for sending selector names to the framework. Or we could do the selector name -> Intent name translation in the engine.

Reply:
I wasn't sure if this approach was viable, so I didn't want to spent time translating selectors to generic names. One approach would be to map selectors in engine into something more resembling intents that we have (ie. with collapse/forward attributes). I'll give it a try.

Reply:
Or we can rename `performIntent` to `performSelector`. Also it's a bit strange that the intents are sent via the focus system despite that we already have an established `TextInputConnection`.

Reply:
My gut says that the names sent from the engine should stay NSStandardKeyBindingResponding names.  I imagine a developer might want to handle a specific NSStandardKeyBindingResponding command and we don't want to hide that from them.  That's my thought.

Reply:
> Or we can rename `performIntent` to `performSelector`. Also it's a bit strange that the intents are sent via the focus system despite that we already have an established `TextInputConnection`.

We had a discussion about this with @matthew-carroll and he made same point.  There could be `performSelector` method on TextInputClient, which could, if needed, translate it to an Intent and pass it to focus system. That way it would also be possible to handle actions that flutter text field is not interested in.